lsp-completion.el

```diff
@@ -213,16 +213,16 @@ Return `nil' when fails to guess prefix."
 (defun lsp-completion--to-internal (items)
   "Convert ITEMS into internal form."
   (--> items
-       (-map (-lambda ((item &as &CompletionItem
-                             :label
-                             :filter-text?
-                             :_emacsStartPoint start-point
-                             :score?))
-               `(:label ,(or filter-text? label)
-                 :item ,item
-                 :start-point ,start-point
-                 :score ,score?))
-             it)))
+    (-map (-lambda ((item &as &CompletionItem
+                          :label
+                          :filter-text?
+                          :_emacsStartPoint start-point
+                          :score?))
+            `(:label ,(or filter-text? label)
```

Comment:
I think the agreement is to have plist on the same level, lets reformat that once @leungbk implements dev formatting as he mentioned in https://github.com/emacs-lsp/lsp-mode/pull/2689#issuecomment-792293282

Reply:
I've moved the formatting stuff to a separate mode and I might need some time to iron out the kinks. I'm fine with leaving the indentation like this until I have that mode ready.

Reply:
@leungbk - given the fact that it seems like there is only one place that is affected apparently after lsp-protocol.el migration I wonder whether it makes sense to invest effort in that. WDYT?